index.html

```diff
@@ -0,0 +1,142 @@
+      <div class="bar bar3"></div>
+    </div>
+    <div class="nav-social">
+      <a href="" id="nav-social-icon-1" class="toggle1"><img alt="facebook-social" src="Images/nav-f.png"></a>
+      <a href="" class="toggle1"><img alt="twitter-social" src="Images/nav-t.png"></a>
+      <a href="" class="toggle1"><h6>English</h6></a>
+      <a href="" class="toggle1"><h6>My Page</h6></a>
+      <a href="" id="nav-social-h63" class="toggle1"><h6 >Logout</h6></a>
+    </div>
+    <nav class="nav-bar">
+      <a href="index.html" class="nav-bar-a1 toggle1" ><img alt="logo" class="nav-bar-image" src="Images/footer-logo.png"></a>
+      <a href="about.html" class="toggle1"><h5 class="red-color">About</h5></a>
+      <a href="#program" class="toggle1"><h5>Program</h5></a>
+      <a href="#join" class="toggle1"><h5>Join</h5></a>
+      <a href="#sponsor" class="toggle1"><h5>Sponsor</h5></a>
+      <a href="" class="toggle1"><h5>News</h5></a>
+      <a href="" class="nav-bar-h56 toggle1"><h5 class="red-color">SC Campaign</h5></a>
+    </nav>
+    
+  </header>
+
+  <section class="intro-text">
+    <h3 id="intro-h3">"Hello! Sharing world"</h3>
+    <h1>SAVE CAUVERY 2022</h1>
+    <p id="para-border">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Esse voluptas quas magnam reprehenderit incidunt! Similique optio in aliquid autem aut voluptate velit nihil iure molestias!</p>
+    <p id="para-time">2022.11.21(THU) ~ 22(FRI)</p>
+    <p id="para-place">@ Cauvery River Karnataka, Tamilnadu</p>
+  </section>
+
+  <section class="main-program" id="program">
+    <h2 id="main-program-header">Main Program</h2>
+    <hr id="main-program-hr"/>
+    <div class="main-program-container">
+      <div class="main-program-item">
+        <img alt="lecture" src="Images/mpi-icon1.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img alt="cc-exhibition" src="Images/mpi-icon2.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img alt="forum" src="Images/mpi-icon3.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img alt="workshop" src="Images/mpi-icon4.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img alt="cc-ignite" src="Images/mpi-icon5.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+    </div>
```

Comment:
As requested by the previous reviewer, kindly ensure that your pages should look almost identical that of the provided design as much as possible. All page elements have similar fonts, colors, spacing, borders, shadows, and paddings than the orignal design. This will help you become a good developer who understands and provides the best services to customers/clients. Below are a few differences spotted that I would like to bring to your attention. 

 Your design            |  The actual design 
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-25 13-09-14](https://user-images.githubusercontent.com/65233860/203993634-d91b9ece-2868-4b15-84a2-d4271634188f.png)  |  ![Screenshot from 2022-11-25 13-08-56](https://user-images.githubusercontent.com/65233860/203993661-3a8449a1-9917-4444-a76c-d3570c42f8e9.png)


 Your design            |  The actual design 
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-25 13-09-34](https://user-images.githubusercontent.com/65233860/203994100-95626a68-16c8-4e51-b2ea-faf4a67c35a1.png)  |  ![Screenshot from 2022-11-25 13-09-27](https://user-images.githubusercontent.com/65233860/203993727-321831ab-0fad-4e3f-897c-58ffd9d30f7f.png)


 Your design            |  The actual design 
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-25 13-09-49](https://user-images.githubusercontent.com/65233860/203993854-423fff7e-c4a6-4e9f-834b-a7702421bf35.png)  |  ![Screenshot from 2022-11-25 13-09-43](https://user-images.githubusercontent.com/65233860/203993887-1012dd88-fa6b-4601-9e3e-63b05241d4d6.png)

Reply:
I understand the speaker's background and main program height but what is wrong is the intro section, I am not sure. I see the red color on both sides, I see the font size is almost similar, I don't think this is a Figma design where it could be exactly the same, you don't expect me to take a scale and measure and calculate it relatively. The number of words in a paragraph is not important though.

Reply:
Please point out the exact changes required, I cannot understand it from the pictures you posted.